<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CentroidPlotsConfigForm</class>
 <widget class="QWidget" name="CentroidPlotsConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="useAutoScaleXCheckBox">
       <property name="text">
        <string>Autoscale X</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="minXLimitLabel">
       <property name="text">
        <string>Minimum X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="minXLimitLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="maxXLimitLabel">
       <property name="text">
        <string>Maximum X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maxXLimitLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="useAutoScaleYCheckBox">
       <property name="text">
        <string>Autoscale Y</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="minYLimitLabel">
       <property name="text">
        <string>Minimum Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="minYLimitLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="maxYLimitLabel">
       <property name="text">
        <string>Maximum Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="maxYLimitLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="numHistoBinsLabel">
       <property name="text">
        <string>Num Histogram Bins:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numHistoBinsLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
